moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -271,7 +213,96 @@ void PlanningSceneMonitor::initialize(const planning_scene::PlanningScenePtr& sc
 
   // start executor on a different thread now
   private_executor_thread_ = std::thread([this]() { private_executor_->spin(); });
-  // reconfigure_impl_ = new DynamicReconfigureImpl(this);
+
+  // Set up publishing parameters
+  rcl_interfaces::msg::ParameterDescriptor desc;
```

Comment:
One of the lines below is causing a seg-fault, I wasn't able to determine which one but commenting them seems to fix it

Reply:
Did you rebase onto https://github.com/ros-planning/moveit2/pull/257? It could be the same timing issue

Reply:
fixed parameter 44c1643